test/integration/flatten.test.js

```diff
@@ -5,58 +5,53 @@
 
 'use strict'
 
-const tap = require('tap')
-const flatten = require('../../lib/util/flatten')
+const test = require('node:test')
```

Comment:
I encountered one of these. This isn't really an integration. Do you mind moving this to `test/unit/util`?